LDAP/Init.hsc

```diff
@@ -106,6 +107,15 @@ ldapSimpleBind ld dn passwd =
            return ()
                          )))
 
+{- | Bind with SASL external mechanism. -}
```

Comment:
Renamed to `simple`. Terms `simple` and `external` are described in LDAP and SASL docs :)